ipaserver/plugins/user.py

```diff
@@ -366,6 +368,13 @@ class user(baseuser):
             },
             'default_privileges': {'PassSync Service'},
         },
+        'System: Manage User Certificate Mappings': {
```

Comment:
The permissions for Users are finer-grained than the permissions for stage users. For instance, for users, there is "System: Modify Users" (which does not grant write on usercertificate) and "System: Manage User Certificates".
For stage user, there is only "System: Modify stage User" which applies to all the attributes.

I applied the same strategy (finer-grained for users) but if it is a mistake I can create a similar permission for cert mapping on stage users.